clients/lsp-csharp.el

```diff
@@ -246,6 +246,11 @@ using the `textDocument/references' request."
       (lsp-show-xrefs (lsp--locations-to-xref-items locations-found) nil t)
     (message "No references found")))
 
+(defun lsp-csharp--handle-os-error (_workspace params)
+  "Handle the 'o#/error' (interop) notification with PARAMS by displaying a message with lsp-warn."
+  (-let (((&hash "FileName" filename "Text" text) params))
```

Comment:
we should not use &hash because soon we will switch to plist for internal representation of the jsonrpc objects.

You have 2 options:

1. Use lsp-get
2. Define data structures and then use the getters/destructorings generated for the interface.

Reply:
done (with opt 2)